src/core_decision_flask_app.py

```diff
@@ -167,6 +168,38 @@ def getStatusDetails():
         logger.error(f'Internal Server Error: {e}')
         abort(500)
 
+
+@app.route('/batch_run_analytics_by_date', methods=['GET'])
+def batch_run_analytics_by_date():
+
+    try:
+        date = request.args.get(DATE_PARAMETER)
+
+        # Return 404 (Not Found) if the record IDs are invalid
+        if (date is None) :
+            logger.error('Missing parameters : '+DATE_PARAMETER+' : '+str(date))
+            abort(404)
```

Comment:
Comment - @Gaurav-Ramakrishna @upkarlidder 

Should this error be 
* 400 - bad request (@Gaurav-Ramakrishna says this)
* 422 - Unprocessable Entity

FYI @JSegrave-IBM